server/server/views/home.py

```diff
@@ -0,0 +1,20 @@
+from cornice.resource import resource, view
+from pyramid.security import Allow, Everyone, ALL_PERMISSIONS
+
+
+@resource(path='/', renderer='json', cors_origins=('http://localhost:3000',))
+class Home(object):
+
+    def __init__(self, request, context=None):
+        self.request = request
+        self.context = context
+
+    def __acl__(self):
+        return [(Allow, Everyone, ALL_PERMISSIONS)]
+
+    @view(permission=ALL_PERMISSIONS)
```

Comment:
Useless. We don't need any requests on the home page and this code also do nothing useful.